shared/src/business/dto/cases/ConsolidatedCaseSummary.ts

```diff
@@ -28,10 +27,10 @@ export class ConsolidatedCaseSummary extends JoiValidationEntity {
 
   constructor(rawCase: any) {
     super('ConsolidatedCaseSummary');
+
     this.caseCaption = rawCase.caseCaption;
     this.docketNumber = rawCase.docketNumber;
     this.docketNumberWithSuffix = rawCase.docketNumberWithSuffix;
-    this.entityName = rawCase.entityName;
```

Comment:
Nice catch!

Are these ConsolidatedCaseSummary records ever stored in dynamo?

Reply:
No, they are not. We double checked by looking at the updateCase persistence method and verified the property consolidatedCases is in the list of fields to omit before persisting.

Reply:
👍